dokka-runners/dokka-gradle-plugin/src/main/kotlin/internal/PluginFeaturesService.kt

```diff
@@ -249,6 +265,18 @@ internal abstract class PluginFeaturesService : BuildService<PluginFeaturesServi
         private const val K2_ANALYSIS_NO_WARN_FLAG_PRETTY =
             "$K2_ANALYSIS_ENABLED_FLAG.noWarn"
 
+        @Suppress("ObjectPrivatePropertyName")
+        private val `To learn about migrating read the migration guide` = /* language=text */ """
```

Comment:
minor: that's an interesting usage of backticks in functions :)
But, IMO, it's an abuse and it will be better to just inline documentation. I don't think we will change it afterwards anyway after release

Reply:
It was a hassle trying to keep the messages in-sync, and using a variable for the repeated parts helped minimise mistakes.

Extracting the repeated parts into variables also helps highlight the differences between the messages.